static/js/stream_edit.js

```diff
@@ -108,11 +97,6 @@ exports.update_stream_description = function (sub) {
     stream_settings.find('.stream-description-editable').html(sub.rendered_description);
 };
 
-exports.prepend_subscriber = function (sub_row, email) {
-    var list = get_subscriber_list(sub_row);
-    list.prepend(format_member_list_elem(email));
-};
```

Comment:
I think the purpose of this was to make sure that you appear at the top if you just subscribed yourself...

Reply:
Yeah, but it wasn't working. Added it's fix.